packages/i18n/src/index.spec.mjs

```diff
@@ -1,94 +0,0 @@
-import fs from 'fs';
-
-import { jest } from '@jest/globals';
-
-describe('i18n', () => {
-	let normalizeI18nInterpolationsTask;
-
-	beforeEach(async () => {
-		jest.spyOn(fs, 'readdirSync').mockReturnValue(['en.i18n.json']);
-		jest.spyOn(fs, 'writeFileSync').mockImplementation(() => {});
-		jest.spyOn(fs, 'readFileSync').mockImplementation(() => JSON.stringify({}));
-		jest.spyOn(fs, 'existsSync').mockReturnValue(true);
-		jest.spyOn(fs, 'rmSync').mockImplementation(() => {});
-	});
-
-	afterEach(() => {
-		jest.resetModules();
-	});
-
-	describe('normalizeI18nInterpolationsTask', () => {
-		it('should return the same object if there are no interpolations', async () => {
-			const i18n = await import('./index.mjs');
-			normalizeI18nInterpolationsTask = i18n.normalizeI18nInterpolationsTask;
-			const fileContents = { test: 'test' };
-			expect(normalizeI18nInterpolationsTask(fileContents, 'en')).toEqual(fileContents);
-		});
-
-		it('should return the replace the interpolation(__) keys to the desired one ({{}})', async () => {
-			const i18n = await import('./index.mjs');
-			normalizeI18nInterpolationsTask = i18n.normalizeI18nInterpolationsTask;
-			const fileContents = {
-				shouldChange: 'this is the __key__',
-				shouldNotChange: 'normal one',
-				multipleKeys: 'sentence __with__ multiple __keys__',
-			};
-			expect(normalizeI18nInterpolationsTask(fileContents, 'en')).toEqual({
-				shouldChange: 'this is the {{key}}',
-				shouldNotChange: 'normal one',
-				multipleKeys: 'sentence {{with}} multiple {{keys}}',
-			});
-		});
-
-		it('should write a txt file with the collected statistics', async () => {
```

Comment:
any reason for this test to be removed?

Reply:
This is not on a file anymore, it's now on the console 👀 on build